plugins/GSdx/res/tfx.fx

```diff
@@ -92,6 +92,7 @@ cbuffer cb0
 	float4 VertexScale;
 	float4 VertexOffset;
 	float2 TextureScale;
+	float4 TextureOffset;
```

Comment:
>Won't be possible

If you say so. :P It would require `vs_cb.TextureScale = GSVector4(1.0f / 16) / WH.xyxy();` to be replaced by something like:
```
		vs_cb.Texture_ScOff.x = (1.0f / 16) / WH.x;
		vs_cb.Texture_ScOff.y = (1.0f / 16) / WH.y;
```